ethportal-api/src/types/discv5.rs

```diff
@@ -24,15 +23,15 @@ pub struct KBucketsTable {
 #[serde(rename_all = "camelCase")]
 pub struct NodeInfo {
     pub enr: Enr,
-    pub node_id: String,
+    pub node_id: NodeId,
```

Comment:
Might be worth adding a comment here that `NodeId` gets properly serialized as a 0x-prefixed hexstring

Reply:
I don't think that's necessary. Since `Serialize/Deserialize` is derived, one should assume that serialization is done correctly.

But if you feel strong about it, I can add it.